kornia/augmentation/container/patch.py

```diff
@@ -376,6 +357,21 @@ def forward_by_params(
             _input = _input.reshape(in_shape)
         return _input, label
 
+    def inverse(  # type: ignore
```

Comment:
Signature differs in overridden method
https://deepsource.io/gh/kornia/kornia/run/3492ca94-6546-4722-8a23-f570db6834d1/python/PYL-W0222